src/app/pages/brows/brows.component.ts

```diff
@@ -9,15 +13,27 @@ import { IProduct } from '@shared/types';
 })
 export class BrowsComponent implements OnInit {
   products: IProduct[] = [] as IProduct[];
+  private unsubscribe$ = new Subject<void>();
 
   constructor (
     private productsService: ProductsService,
   ) {}
 
   ngOnInit(): void {
-    this.productsService.fetchData('eyebrow')
-    .subscribe(result => {
-        this.products = result;
+    fetchProductProps([EYEBROW_PRODUCT], this.productsService)
+    .pipe(takeUntil(this.unsubscribe$))
+    .subscribe({
+      next: productProps => {
```

Comment:
Just a FYI

Different projects have different rules, but often in the lint settings you can see the rule (no any type or no empty type)
Therefore, it’s better to immediately accustom yourself to prescribing types everywhere, even if it’s just void, it will be easier for a potential reviewer at least